client/src/components/TaskDetails/TaskDetails.js

```diff
@@ -2,17 +2,40 @@ import React from "react";
 import styles from "./TaskDetails.module.css";
 import { PropTypes } from "prop-types";
 
-const TaskDetails = ({ closeModalHandler }) => {
+const TaskDetails = ({
+  closeModalHandler,
+  title,
+  location,
+  description,
+  startDateTimeString,
+  endDateTimeString,
+}) => {
   return (
     <div className={styles.modal}>
+      <div className={styles.overlay}></div>
       <div className={styles.modal}>
-        <div className={styles.overlay}></div>
-        <div className={styles.modal}>
-          <button className={styles.closeModal} onClick={closeModalHandler}>
-            close
-          </button>
-          <h1>Task Details</h1>
-          <p>... Task Description ...</p>
+        <button className={styles.closeModal} onClick={closeModalHandler}>
+          close
```

Comment:
For consistency I would just capitalize the `C` in close

Reply:
Fixed.